interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/out/FaultOutboundInterceptorTest.java

```diff
@@ -77,17 +57,17 @@ class FaultOutboundInterceptorTest extends AbstractSdtUnitTestBase {
     void testHandleMessage() {
         SdtContext.getContext().setServiceRequestId(1L);
         final SoapMessage soapMessage = getDummySoapMessageWithFault();
-        final FaultOutboundInterceptor faultOutboundInterceptor = new FaultOutboundInterceptor(mockServiceRequestDao);
+        final FaultOutboundInterceptor faultOutboundInterceptor = new FaultOutboundInterceptor(requestDaoService);
         final ServiceRequest serviceRequest = new ServiceRequest();
         when(mockServiceRequestDao.fetch(ServiceRequest.class, 1L)).thenReturn(serviceRequest);
         mockServiceRequestDao.persist(serviceRequest);
```

Comment:
This is actually calling the persist() method on mockServiceRequestDao.  It's unlikely that the test needs to do this and it's probably a hang over from Easy Mock.  This may need to be changed to `verify(mockServiceRequestDao).persist(serviceRequest)`.

Reply:
To clarify, this line is actually invoking the persist() method on the mockServiceRequestDao object.  It is unlikely that the test needs to do this directly - it's more likely that invoking the handleMessage() will cause this to be invoked and you can then verify that it has been invoked.  You should be able to remove this line and change the verify() below to only check for one invocation of it, i.e. `verify(mockServiceRequestDao).persist(serviceRequest);`